taxonomy/utils.py

```diff
@@ -53,86 +53,36 @@ def get_whitelisted_serialized_skills(course_key):
     return skills_data
 
 
-def update_course_skills_data(course_key, skill_external_id, confidence, skill_data):
+def update_skills_data(key_or_uuid, skill_external_id, confidence, skill_data, is_programs):
```

Comment:
Suggestion: we can replace `is_programs` to `product_type` and then check if `product_type` is `Program` or any other. This way we can scale this up for multiple products as well